### PR TITLE
Add Charateristic in ProfilingResult

### PR DESCRIPTION
**Pull request by KangJH**
## Main changes
- Add characteristics field in ProfilingResult.

### CODE REVIEW

File: type.proto [hunk 1]
```diff
@@ -77,6 +77,14 @@ message Object {
 }
 
 message ProfilingResult {
+  message Characteristic {
+    oneof member_oneof {
+      uint64 size = 1;
+      float decimal = 2;
+      string text = 3;
+    }
```
Comment (sebastian-seoul-robotics) [inline type.proto:85]: this is the definition of a characteristic: `std::variant<int64_t, double, std::string>`. So I think this does not match with this protobuf definitions

File: type.proto [hunk 2]
```diff
@@ -77,6 +77,14 @@ message Object {
 }
 
 message ProfilingResult {
+  message Characteristic {
+    oneof member_oneof {
+      uint64 size = 1;
```
Comment (sebastian-seoul-robotics) [inline type.proto:82]: we are mostly using it for size values e.g. (number of points) but you can input any integer value (also negative)

Comment (KangJH) [reply type.proto:82]: then, variable name `integer` is OK?
 ```
message Characteristic {
    oneof member_oneof {
      int64 integer = 1;

Comment (sebastian-seoul-robotics) [reply type.proto:82]: Puuh. `integer` sounds to me not like a name should give a variable. Maybe `integer_number` and `decimal_number`?

Comment (KangJH) [reply type.proto:82]: OK, I will go with that.

File: type.proto [hunk 3]
```diff
@@ -77,6 +77,14 @@ message Object {
 }
 
 message ProfilingResult {
+  message Characteristic {
+    oneof member_oneof {
+      int64 integer_number = 1;
+      float decimal_number = 2;
```
Comment (sebastian-seoul-robotics) [inline type.proto:83]: should'nt it be `double`?

Comment (KangJH) [reply type.proto]: Done!

**Review by sebastian-seoul-robotics — APPROVED** [review]
:beers:

**Pull request merged**